Entity/Mail.php

```diff
@@ -197,7 +206,15 @@ public function getMessage()
             ->setBcc($this->getBcc())
             ->setReplyTo($this->getReplyTo())
             ->setBody($this->getBody(), 'text/html');
-
+        $messageHeaders = $message->getHeaders();
+        $dbHeaders = $this->getHeaders();
+        if (count($dbHeaders)) {
```

Comment:
no need to count I think right ?

Reply:
It was failing when there were no headers in db. Changed failback to an array casting in getter method.